tests/test_basic_example_v2.py

```diff
@@ -63,6 +63,16 @@ def testmodulev2_venv(pytestconfig) -> env.VirtualEnv:
         # set up environment
         venv: env.VirtualEnv = env.VirtualEnv(env_path=venv_dir)
         venv.init_env()
+        # workaround for pypa/build#405: unset PYTHONPATH because it's not required in this case and it triggers a bug in build
+        with open(os.path.join(venv.site_packages_dir, "sitecustomize.py"), "a") as fd:
```

Comment:
You have to use `usercustomize.py` here, because some Linux distribution already set a `sitecustomize.py` by default, causing this `sitecustomize.py` to be ignored. That would break the pytest-docker tests suite.

Reply:
This line appends to `sitecustomize.py`, if it exists, wouldn't that work?

Reply:
No. This append happens in the `sitecustomize.py` file of the venv. The `sitecustomize.py` I was talking about is defined in the python system library. So that trick wouldn't work.